engine/class_modules/sc_hunter.cpp

```diff
@@ -4838,6 +4867,17 @@ struct kill_command_t: public hunter_spell_t
         flankers_advantage.proc -> occur();
         cooldown -> reset( true );
         p() -> buffs.strength_of_the_pack -> trigger();
+
+        if ( p() -> options.t28_2pc )
+        {
+            if ( rng().roll( 0.4 ) )
+            {
+              p() -> cooldowns.wildfire_bomb -> reset( true );
```

Comment:
Judging by the wording this is wrong. It does not reset the cd but supresses triggering the cd while the 4pc buff is up. So this should be done by overriding `action_t::update_ready()` and skipping it in case the buff's up.

Reply:
Smth like
```cpp
void update_ready( timespan_t cd_duration ) override {
  if ( p() -> buffs.mad_bombardier -> check() )
    return; // maybe sim->print_debug() smth about mad bombardier 

  hunter_spell_t::update_ready( cd_duration );
} 
```

Reply:
Fixed